az-digital/az_quickstart
# Closes #2197 use card title field in demo content, Closes #2426 Add view paragraphs to demo content.

## Description
Added new page, and menu item for news views paragraphs.
Added 3 new paragraphs with view references
Update card deck migration to use field_az_title, which was added to quickstart.

## Related issues
Closes #2197 
Closes #2426 

## How to test
Go to /pages/news-view-paragraphs
See views embedded in a page via views paragraph

See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/card-decks
See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/news-view-paragraphs
See https://8d3d666c-b5e0-4a90-95cc-308918134ee6.probo.build/pages/person-view-paragraphs


Existing card deck page https://demo.quickstart.arizona.edu/pages/card-decks

Go to /pages/card-decks
And see new card deck group titles


## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
